Tests/LeafKitTests/GHTests/VaporLeaf.swift

```diff
@@ -50,21 +50,14 @@ final class GHLeafIssuesTest: XCTestCase {
     func testGH105() throws {
         do {
             let template = """
-            #if(1 + 1 == 2):hi#endif
+            #if((1 + 1) == 2):hi#endif
             """
             let expected = "hi"
             try XCTAssertEqual(render(template, ["a": "a"]), expected)
         }
         do {
             let template = """
-            #if(2 == 1 + 1):hi#endif
-            """
-            let expected = "hi"
-            try XCTAssertEqual(render(template, ["a": "a"]), expected)
-        }
-        do {
-            let template = """
-            #if(1 == 1 + 1 || 1 == 2 - 1):hi#endif
```

Comment:
Why is this test removed?

Reply:
moved it to `LeafParserTests.testGroupingTwo` to make it more clear that this is testing operator precedence